application/controllers/ScheduleController.php

```diff
@@ -4,16 +4,59 @@
 
 namespace Icinga\Module\Notifications\Controllers;
 
+use Icinga\Module\Notifications\Common\Database;
+use Icinga\Module\Notifications\Common\Links;
 use Icinga\Module\Notifications\Forms\EntryForm;
 use Icinga\Module\Notifications\Forms\ScheduleForm;
+use Icinga\Module\Notifications\Model\Schedule;
+use Icinga\Module\Notifications\Widget\Calendar\Controls;
 use Icinga\Module\Notifications\Widget\RecipientSuggestions;
+use ipl\Html\Form;
 use ipl\Html\Html;
+use ipl\Stdlib\Filter;
 use ipl\Web\Compat\CompatController;
 use ipl\Web\Url;
+use ipl\Web\Widget\ButtonLink;
 
 class ScheduleController extends CompatController
 {
-    public function indexAction()
+    public function indexAction(): void
+    {
+        $id = (int) $this->params->getRequired('id');
+
+        $query = Schedule::on(Database::get())
+            ->filter(Filter::equal('schedule.id', $id));
+
+        /** @var ?Schedule $schedule */
+        $schedule = $query->first();
+        if ($schedule === null) {
+            $this->httpNotFound(t('Schedule not found'));
+        }
+
+        $this->addTitleTab(sprintf(t('Schedule: %s'), $schedule->name));
+
+        $this->controls->addHtml(
+            (new ButtonLink(
+                null,
+                Links::scheduleSettings($id),
+                'cog'
+            ))->openInModal()
+        );
+
+        $calendarControls = (new Controls())
+            ->setAction(Url::fromRequest()->getAbsoluteUrl());
+        if ($this->getRequest()->getHeader('X-Icinga-Container') === 'modal-content') {
+            $this->getResponse()->setHeader('X-Icinga-Modal-Layout', 'wide');
+            $calendarControls->setBaseTarget('modal-content');
+        }
```

Comment:
This is/was only for testing purposes which shouldn't have been introduced to `main` to begin with. Remove this please.